packages/teams-js/src/public/visualMedia.ts

```diff
@@ -0,0 +1,308 @@
+import { sendAndHandleSdkError } from '../internal/communication';
+import { ensureInitialized } from '../internal/internalAPIs';
+import { errorNotSupportedOnPlatform, FrameContexts } from './constants';
+import { DevicePermission } from './interfaces';
+import { runtime } from './runtime';
+
+/**
+ * Interact with image and video. It lets the app developer ask the user to get images or video from their camera / camera roll / file system.
+ * @beta
+ */
+export namespace visualMedia {
+  /**
+   * @hidden
+   * Hide from docs
+   * --------
+   * All properties common to Image and Video Props
+   * @beta
+   */
+  interface VisualMediaProps {
+    /**
+     * max limit of media allowed to be selected in one go, current max limit is 10.
```

Comment:
>current max limit is 10. [](http://example.com/codeflow?start=60&length=24)

To avoid using the word max twice with slightly different meanings here, consider saying "Only values <= 10 are supported" rather than "current max limit is 10."

Reply:
Thanks for your suggestion! Updated.